jaeger-crossdock/src/test/java/com/uber/jaeger/crossdock/resources/behavior/http/TraceBehaviorResourceTest.java

```diff
@@ -82,12 +82,10 @@ public static void setUpLogger() {
 
   @Before
   public void setUp() throws Exception {
-    port = System.getenv("TRACE_TEST_BEHAVIOR_PORT");
```

Comment:
I did not find any usage and this test is not bundled in jar so it's not used in the container.